copyblogger/genesis-simple-hooks
# Suggestion: remove old plugin.php bootstrap file

There are currently two plugin bootstrap files. This PR moves the additional header rows from the old plugin bootstrap file to the preferred file (which follows the convention of using a file with the same name as the plugin/plugin folder).

Additionally, the unneeded `plugin.php` file is deleted.

---

Note that there is no minimum WP version indicated in this header.

I am not sure when WP started expecting the header to live in a file matching the plugin/plugin folder name, but it's been that way as long as I can recall.

My concern is that, previously, that bootstrap file was included in `plugin.php`, and I am not 100% about very old versions of WP possibly expecting that as a convention.

---

### How to test

1. Deactivate your existing install of genesis-simple-hooks
2. Activate this version
3. Make sure activation was successful
